.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Give users an ignored place to build
 /build/*
+
+# Don't care about vim swap files
+*.swp
```

Comment:
I think editor specific things like this belong in your personal `~/.gitignore`, not in the project itself.

Reply:
:exploding_head: `~/.gitignore` is a thing!? That's so useful! Thanks

Reply:
...it's not a thing unless you also run `git config --global core.excludesfile ~/.gitignore`. but I've now done that.